### PR TITLE
Retrieves precomputed scores

### PR DESCRIPTION
**Pull request by tkmamidi**
This PR is a refactor of how the webapp retrieves/displays DITTO scores for variants.

- [x] Loads tabix indexed files from [UAB LTS S3](https://s3.lts.rc.uab.edu/cgds-public/dittodb/dittodb.html) using pysam
- [x] Retrieves variants based on chromosome and position
- [x] Streamlit app to display variants in a table format

### CODE REVIEW

**Review by SeriousHorncat — CHANGES_REQUESTED** [review]
- Followed README instructions for environment setup and deployment.  Successfully  deployed within the `ditto-env` virtual environment on MaxOSX.

Screenshot 2024-10-21 at 12 37 32 PM

Failing to to query DITTO scores using the local deployment for each of the examples listed above in the UI

Screenshot 2024-10-21 at 12 40 07 PM

Screenshot 2024-10-21 at 12 42 08 PM

**Comment by tkmamidi** [comment]
Thanks for the review! 
This seems like a known issue with pysam. This Gihub issue suggests to export the certs path so pysam can reference the correct path - https://github.com/pysam-developers/pysam/issues/1257#issuecomment-2102429054

To address this, I added the path to certs as environment variable that works (in general and on streamlit cloud). Is it possible for you to check the path and see if certs are available? 
Here's the path I used -
https://github.com/uab-cgds-worthey/DITTO-UI/blob/f03fab1a324154ff007375b82746397e36f9d3aa/src/app.py#L24C35-L24C69

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (SeriousHorncat) [inline .devcontainer/devcontainer.json:1]: Is there a reason to include the devcontaine.json file with the project in GitHub? or is this something that accidentally made it into the branch.

**Review by SeriousHorncat — COMMENTED** [review]
Can confirm it works on my macosx when commenting out the line 24  in `app.py` works locally.

    # os.environ['CURL_CA_BUNDLE']='/etc/ssl/certs/ca-certificates.crt'

We can chat in standup tomorrow as a team to brainstorm solutions. I do recommend us coming up with a solution that works for local and deployed setups.  That way if some pulls your project and wants to run it locally on a variety of setups, folks will be able to use Ditto locally.

**Comment by tkmamidi** [comment]
Added dockerfile and compose to deploy locally !

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (tkmamidi) [reply .devcontainer/devcontainer.json]: Now that we have dockerfile, deleted this file from the repo.

**Review by SeriousHorncat — APPROVED** [review]
Works like a charm with the revised README and docker approach for deployment. 👍 Is very easy to get it up and running.

image

**Pull request merged**